### ISSUE TITLE
Inability to change libvlc settings

### ISSUE THREAD

**Issue by Ivshti** [RSATom/QmlVlc#62]
Besides what is exposed through the QmlVlcConfig class, there's nothing else.

Use case - I want to set `--no-osd`, `--no-media-library`

Would you consider a way of exposing the opts vector or adding an ability to push to it with an external func?

**Comment by RSATom** [comment]
Good point. Something similar was not added yet just because this module was used in Browser Plugin mostly, which have to be security restricted...
